__test__/api/admins/getSingleAdmin.test.js

```diff
@@ -0,0 +1,63 @@
+const app = require("../../../server");
+const supertest = require("supertest");
+const mongoose = require("mongoose");
+const AdminModel = require("../../../models/admins");
+const { getOrgToken } = require("../../../utils/auth/tokenGenerator");
+const { validateOrg } = require("../../../utils/validationRules/admins/getSingleAdminSchema");
+
+const request = supertest(app);
+
+describe("READ /v1/admins/get-single-admin", () => {
+  let adminInfo = {
+    adminId: "adminIdStr",
+  };
+  let url = `/v1/admins/get-single-admin`;
+
+  it("gets single admin ", async () => {
+    const res = await request
+      .get(url)
+      .set("Authorization", `bearer ${global.orgToken}`)
+      .send(adminInfo);
```

Comment:
There is no need for admin Info . This is a GET Request. Please refer to the Docs for proper guide

Reply:
You don't also need to add anything to the body of the request